styles/mobile.css

```diff
@@ -70,11 +73,16 @@
     .burger::after {top: 15px;}
     input:checked~ul{
         height: 125px;
-        transition-duration: 500ms;
-        transition-timing-function: ease-in-out;
     }
-        #burger-toggle:checked ~ .burger {
-            transform:translateX(-5px);
-            transform:translateY(-10px);
-            transform:rotate(45deg);
-        }
\ No newline at end of file
+    #burger-toggle:checked ~ .burger {
+        transform:translate(-5px, 10px) rotate(45deg);
+    }
```

Comment:
Do multiple transformations using only ONE declaration, otherwise only the last one applies